website/source/layouts/kubernetes.erb

```diff
@@ -25,7 +25,7 @@
             <li<%= sidebar_current("docs-kubernetes-resource-namespace") %>>
               <a href="/docs/providers/kubernetes/r/namespace.html">kubernetes_namespace</a>
             </li>
-            <li<%= sidebar_current("docs-kubernetes-resource-persistent-volume") %>>
+            <li<%= sidebar_current("docs-kubernetes-resource-persistent-volume-x") %>>
```

Comment:
This is to avoid highlighting this item on `persistent-volume-claim`. I can't say I like this approach (because it makes generating docs tricky), but it fixes the problem.